tests/generic_config_updater/multiasic_generic_updater_test.py

```diff
@@ -0,0 +1,167 @@
+import json
+import jsonpatch
+import unittest
+from unittest.mock import patch, MagicMock
+
+import generic_config_updater.change_applier
+import generic_config_updater.generic_updater
+import generic_config_updater.services_validator
+import generic_config_updater.gu_common
+
+# import sys
+# sys.path.insert(0,'../../generic_config_updater')
+# import generic_updater as gu
+
+class TestMultiAsicPatchApplier(unittest.TestCase):
+
+    @patch('generic_config_updater.gu_common.ConfigWrapper.get_empty_tables', return_value=[])
+    @patch('generic_config_updater.gu_common.ConfigWrapper.get_config_db_as_json')
+    @patch('generic_config_updater.gu_common.PatchWrapper.simulate_patch')
+    @patch('generic_config_updater.generic_updater.ChangeApplier')
+    def test_apply_patch_specific_namespace(self, mock_ChangeApplier, mock_simulate_patch, mock_get_config, mock_get_empty_tables):
```

Comment:
Could you add some negative case, for exmple not_fully_udpated or empty tables just like that in single asic appy patch>